pkg/arm/v6/data/master-startup.sh

```diff
@@ -1,5 +1,53 @@
 #!/bin/bash -ex
 
+{{ if gt (len .Config.SecurityPatchPackages) 0 }}
+logger -t master-startup.sh "installing red hat cdn configuration on $(hostname)"
+cat >/var/lib/yum/client-cert.pem <<'EOF'
+{{ CertAsBytes .Config.Certificates.PackageRepository.Cert | String }}
+EOF
+cat >/var/lib/yum/client-key.pem <<'EOF'
+{{ PrivateKeyAsBytes .Config.Certificates.PackageRepository.Key | String }}
+EOF
+cat > /etc/yum.repos.d/kickstart.repo <<'EOF'
+[rhel-7-server-rpms]
+name=Red Hat Enterprise Linux 7 Server (RPMs)
+baseurl=https://cdn.redhat.com/content/dist/rhel/server/7/7Server/$basearch/os
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslcacert=/etc/rhsm/ca/redhat-uep.pem
+sslclientcert=/var/lib/yum/client-cert.pem
+sslclientkey=/var/lib/yum/client-key.pem
+enabled=yes
```

Comment:
Should we keep them disabled by default and as part of the script enable? Just thinking...